canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -136,6 +136,11 @@ export default class PropertiesController {
 		return this.propertiesStore.getActiveTab();
 	}
 
+	// Set the top-level active tab using tabId
+	setTopLevelActiveGroupId(tabId) {
```

Comment:
@srikant-ch5 can you add something in the controller action in the test harness so this can be tested?
![Screenshot 2024-02-08 at 10 54 16 AM](https://github.com/elyra-ai/canvas/assets/21126000/5a037948-b550-41cb-a535-40dd764df3d1)

Reply:
Hi @matthoward366 I have added setActiveTab in Controller Action to test. Please find below recording of the same.


https://github.com/elyra-ai/canvas/assets/16302714/e99a7fdb-407a-473b-b953-41a742ef4bd2